docs/standard-input-output.adoc

```diff
@@ -0,0 +1,64 @@
+:page-title: Standard input/output
+:page-description: JUnit Jupiter extension for simulating standard input or catching standard output.
+
+The `StdIOExtension` adds a simple way to test your classes that read from the standard
+input (`System.in`) or write to the standard output (`System.out`).
+
+== How to use the extension
+
+If you would like to use the special parameters provided by the extension
+just annotate your test class with `@ExtendWith(StdIOExtension.class)`.
+Then, when writing your tests, you can pass the following two classes as arguments (provided by the extension):
+
+StdIn(`org.junitpioneer.jupiter.StdIOExtension.StdIn`)::
+This is for simulating input from the standard in. If you are using this parameter, you have to
+annotate you method with `@StdInSource`. This annotation takes a String array which holds the values that
+you would like to read instead of waiting for input on the standard input. Each value of the array
+represents one line of input.
+
+The following example shows how to pass the values `Hello` and `World` as standard input:
+
+[source,java]
+----
+    @ExtendWith(StdIOExtension.class)
+    public class StandardInputOutputTests {
+        ...
+    @Test
+    @StdInSource({"Hello", "World"})
+    void test(StdIn in) {
+        ...
+
+----
+
+StdOut(`org.junitpioneer.jupiter.StdIOExtension.StdOut`)::
+This is for catching output going to the standard out. This parameter does not require any additional annotations
+except for the `@ExtendWith(StdIOExtension.class)` on your test class (see below).
+
+[source,java]
+----
+    @ExtendWith(StdIOExtension.class)
+    public class StandardInputOutputTests {
+        ...
+    @Test
+    void test(StdOut out) {
+        ...
+
+----
+
+To check your method under tests interactions with `StdIn` or `StdOut`, you can call
+`capturedLines()` (available in both `StdIn` and `StdOut`) inside your test to verify
+the lines read by `StdIn` or written by `StdOut`.
+This array contains the redirected in- or output as lines.
+
+For example, after calling `System.out.print("Hello")` and `System.out.println("World")`,
+the `capturedLines()` method would contain an array of the String "HelloWorld" because
+the first method does not print a line break.
```

Comment:
```suggestion
For example, after calling `System.out.print("Hello")` and `System.out.println("World")`,
the `capturedLines()` method would contain an array of the String "HelloWorld", because
the first method does not print a line break.
```